ui/src/submissions/literature/containers/__tests__/DataImporterContainer.test.jsx

```diff
@@ -16,34 +16,41 @@ describe('DataImporterContainer', () => {
         initialDataError: { message: 'Import Error' },
       }),
     });
-    const wrapper = mount(
+
+    render(
       <Provider store={store}>
         <DataImporterContainer onSkipClick={jest.fn()} />
       </Provider>
     );
-    expect(wrapper.find(DataImporter)).toHaveProp({
-      isImporting: true,
-      error: fromJS({ message: 'Import Error' }),
-    });
+
+    expect(screen.getByTestId('data-importer-error')).toHaveTextContent(
```

Comment:
nit: you can utilize the `const {getByTestId} = `... and remove the screen if you want